builds/fabric8-che/pom.xml

```diff
@@ -172,6 +190,18 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
Why do you fully skip the dependency analysis ? We have a build options to skip it fully from the command line.
And else there is a more fine-grained way to skip some dependency that might be a problem (for false-positive cases for example).

See here for the way to remove dependency errors individually: https://github.com/redhat-developer/rh-che/blob/07556f724354bf008e0c09587fc44b2e40e7e000/builds/fabric8-che/assembly/pom.xml#L190